src/modsim2/model/utils.py

```diff
@@ -68,3 +71,37 @@ def get_run_from_name(
 
     # Return if so
     return runs[0]
+
+
+def get_run_from_name(
```

Comment:
would argue this isn't from name now? run name is constructed inside the method rather than passed as param. Could just be `get_wandb_run`?

Reply:
Agreed, have changed it